index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var forEach = require('foreach');
+var getprototypeof = require('getprototypeof');
```

Comment:
unfortunately this module does not actually fall back to `Object.getPrototypeOf` when available - and it also relies on `constructor`, which is mutable.